src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -245,8 +245,6 @@
     public static final String ALLOW_FILE_AUTO_OPEN_BROWSE = "allowFileAutoOpenBrowse";
     public static final String CUSTOM_TAB_NAME = "customTabName_";
     public static final String CUSTOM_TAB_FIELDS = "customTabFields_";
-    public static final String USE_UNIT_FORMATTER_ON_SEARCH = "useUnitFormatterOnSearch";
```

Comment:
Probably it makes sense to add a migration to it

Reply:
I believe there is nothing to migrate to. This preference is not used anymore. It was abandoned long ago and I don't know since when. It is no more, has ceased to be and met it's maker. This is an ex preference.